docs/guides/http_crawler.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: http-crawler-guide
+title: "HttpCrawler guide"
+sidebar_label: "HttpCrawler"
+description: Your first steps into the world of scraping with Crawlee
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+
+&#8203;<ApiLink to="http-crawler/class/HttpCrawler">`HttpCrawler`</ApiLink> is our simplest and fastest crawler, as it doesn't do any HTML parsing. This enables us to attach any DOM engine of our choosing.
+
+:::note
+
+For starters, we recommend using <ApiLink to="cheerio-crawler/class/CheerioCrawler">`CheerioCrawler`</ApiLink> instead as it already comes with the old-school jQuery syntax. And it's **fast**.
+
+:::
+
+## How the crawler works
+
+&#8203;<ApiLink to="http-crawler/class/HttpCrawler">`HttpCrawler`</ApiLink> crawls by making plain HTTP requests to the provided URLs using the specialized [got-scraping](https://github.com/apify/got-scraping) HTTP client. The URLs are fed to the crawler using <ApiLink to="core/class/RequestQueue">`RequestQueue`</ApiLink>. The HTTP responses it gets back are usually HTML pages. The same pages you would get in your browser when you first load a URL. But it can handle any content types with the help of the <ApiLink to="http-crawler/interface/HttpCrawlerOptions#additionalMimeTypes">`additionalMimeTypes`</ApiLink> option.
+
+:::info
+
+Modern web pages often do not serve all of their content in the first HTML response, but rather the first HTML contains links to other resources such as CSS and JavaScript that get downloaded afterwards, and together they create the final page. To crawl those, see <ApiLink to="puppeteer-crawler/class/PuppeteerCrawler">`PuppeteerCrawler`</ApiLink> and <ApiLink to="playwright-crawler/class/PlaywrightCrawler">`PlaywrightCrawler`</ApiLink>.
```

Comment:
Small nit since you need one or the other, not both

```suggestion
Modern web pages often do not serve all of their content in the first HTML response, but rather the first HTML contains links to other resources such as CSS and JavaScript that get downloaded afterwards, and together they create the final page. To crawl those, see <ApiLink to="puppeteer-crawler/class/PuppeteerCrawler">`PuppeteerCrawler`</ApiLink> or <ApiLink to="playwright-crawler/class/PlaywrightCrawler">`PlaywrightCrawler`</ApiLink>.
```

Reply:
This is copied from the cheerio counterpart